lib/utils.js

```diff
@@ -219,8 +218,13 @@ var Utils = module.exports = {
   },
 
   toDefaultValue: function(value) {
-    if (lodash.isFunction(value)) {
-      return value();
+    if (typeof value === 'function') {
+      var tmp = value();
+      if (tmp instanceof DataTypes.ABSTRACT) {
```

Comment:
You might also want to check `instanceof` when value is not a function, since default values should be `new()`'ed when the model is define (although they might not be but would be a good future proof to have.